plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorSmokeTest.java

```diff
@@ -180,15 +189,18 @@ public void testSchemaEvolutionOnTableWithColumnInvariant()
                 tableName,
                 getLocationForTable(bucketName, tableName)));
 
-        assertThatThrownBy(() -> query("INSERT INTO invariants VALUES(2)"))
-                .hasMessageContaining("Inserts are not supported for tables with delta invariants");
```

Comment:
How was that working? Shouldn't it be failing with table not found?

Reply:
Not sure if I understand the question correctly. The previous table name was `invariants` having column invariants and it existed, so it shouldn't throw table not found in my opinion. I renamed to a new table name with random suffix to avoid modifying shared tables.